### PR TITLE
Update CAPP Compliance to 2019

### LINKED ISSUE #1946: Introduce CAPP Compliance product page
- [ ] Introduce CAPP Compliance logo
- [ ] Add H1 (inside hero)
- [ ] Add on-page navigation menu, using CAPP Compliance logo/base color
- [ ] Add testimonials (short generic quotes)
- [ ] Add introduction
- [ ] Make summary/introduction visually more attractive (graphic?)
- [ ] Add main content
- [ ] Add CTA to download productsheet
- [ ] Redirect KPP & KM product pages to new product page
- [ ] Remove old KPP & KM productsheets

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
- [x] Introduce CAPP Compliance logo
- [x] Add H1 (inside hero)
- [x] Add on-page navigation, using CAPP Compliance logo/base color
- [x] Add introduction
- [x] Add main content
- [x] Add CTA to download productsheet
- [x] Redirect KPP & KM product pages to new product page
- [x] Remove old KPP & KM productsheets
- [x] Update homepage solutions
- [x] Update main navigation
- [x] Update Product index
- [x] Update all links to `/kwaliteitspaspoort` and `/kwaliteitsmonitor`

Close #1946

### CODE REVIEW

**Comment by RonaldDijkstra** [comment]
Add testimonials out of scope for this PR.

**Comment by RonaldDijkstra** [comment]
When NL -> load product_layout stylesheet
When DE -> load CAPP LMS stylesheet 
When EN -> load (old) KPP stylesheet

Wow

File: data/capp-compliance-testimonials.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+- name: Kristel Vissers
+  company: Projectleider Compliance, Máxima Medisch Centrum
+  avatar: testimonials/kristel-vissers.jpg
+  quote: |
+    Met behulp van Defacto beschikken leidinggevenden over (stuur)informatie en zijn onze (zorg)professionals bewust bezig met hun eigen ontwikkeling en nemen zij hierin nu zelf de verantwoordelijkheid.
+
+- name: Esther Maas-Schelling
+  company: Projectleider, Strukton
+  quote: |
+    Defacto heeft __echt__ met ons __meegedacht__ en geholpen met het borgen van de compliance binnen onze organisatie.
```
Comment (michellusthof) [inline data/capp-compliance-testimonials.yml]: Add extra space after 'meegedacht' and remove bold text.

File: source/localizable/capp-lms-new/_overview.nl.erb [hunk 2]
```diff
@@ -54,7 +54,7 @@
...
       </p>
       <p>
-        Voor elke team-, afdelings- of divisiemanager is er op elk niveau permanent inzicht in de kwaliteit en de vakbevoegdheid van de medewerkers. Dat inzicht complementeren we met onze Kwaliteitsmonitor®, die geaggregeerd inzicht geeft in de performance van de organisatie en die bij uitstek de resultaten biedt die aan de raad van bestuur of directie gerapporteerd kunnen worden. Zo worden accreditaties gemakkelijk en voldoet de organisatie altijd aan de wet- en regelgeving en wordt compliant zijn en blijven onderdeel van het primaire proces.
+        Voor elke team-, afdelings- of divisiemanager is er op elk niveau permanent inzicht in de kwaliteit en de vakbevoegdheid van de medewerkers. Dat inzicht complementeren we met CAPP Compliance, die geaggregeerd inzicht geeft in de performance van de organisatie en die bij uitstek de resultaten biedt die aan de raad van bestuur of directie gerapporteerd kunnen worden. Zo worden accreditaties gemakkelijk en voldoet de organisatie altijd aan de wet- en regelgeving en wordt compliant zijn en blijven onderdeel van het primaire proces.
```
Comment (michellusthof) [inline source/localizable/capp-lms-new/_overview.nl.erb]: "[...]**dat** geaggregeerd inzicht geeft [...]"

File: data/capp-compliance-testimonials.yml (hunk 1 above)
Comment (RonaldDijkstra) [reply data/capp-compliance-testimonials.yml]: There's already a space, so no.

File: source/localizable/capp-lms-new/_overview.nl.erb (hunk 2 above)
Comment (RonaldDijkstra) [reply source/localizable/capp-lms-new/_overview.nl.erb]: this is in this PR?

Comment (RonaldDijkstra) [reply source/localizable/capp-lms-new/_overview.nl.erb]: Oh yes. I see

File: source/localizable/capp-compliance/nl/_product-overview.erb [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        Operationaliseer wet- en regelgeving
+      </h2>
+
+      <p>
+        Ondersteun <strong>wet- en regelgeving</strong> voor uw branche of organisatie. Eenvoudig <strong>vereisten opstellen</strong> voor functie, rol en afdeling, en koppelen aan <strong>concrete leerinterventies</strong>. Onze consultants geven <strong>deskundig advies</strong> op basis van uitgebreide praktijkervaring.
+      </p>
+    </div>
+
+    <div class="col col-img">
+      <%= image_tag "screenshots/capp-compliance/capp-compliance-inzicht-manager.png", title: "Compliance inzicht voor managers", alt: "Compliance inzicht voor managers" %>
```
Comment (michellusthof) [inline source/localizable/capp-compliance/nl/_product-overview.erb]: This image belongs to 'monitor compliance op elk niveau'

File: source/localizable/capp-compliance/nl/_product-overview.erb [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+      </h2>
+
+      <p>
+        Met de overzichten van CAPP Compliance heeft u <strong>altijd inzicht</strong> in alle vereisten, ongeacht uw rol binnen de organisatie. Monitor compliance per <strong>afdeling, functie, rol en een combinatie daarvan</strong>. Zo maakt u zowel intern monitoren als extern rapporteren voor accreditaties efficiënt en gebruiksvriendelijk.
+      </p>
+
+    </div>
+
+    <div class="col col-img">
+      <%= image_tag "screenshots/capp-compliance/capp-compliance-certificaatbeheer.png", title: "Certificaatbeheer", alt: "Certificaatbeheer" %>
```
Comment (michellusthof) [inline source/localizable/capp-compliance/nl/_product-overview.erb]: This image belongs to 'operationaliseer wet- en regelgeving'

**Pull request merged**